src/ucp/api/ucp_def.h

```diff
@@ -771,5 +771,45 @@ typedef struct ucp_ep_params {
  */
 #define UCP_ENTITY_NAME_MAX 32
 
+/**
+ * @ingroup UCP_ENDPOINT
+ * @brief A transport name and device name pair used by this endpoint.
+ */
+typedef struct ucp_transport_entry_t {
```

Comment:
pls remote ucp_transport_entry_t from here (to align with other structs)

Reply:
I'm not sure what you are asking here

Reply:
sorry, remote=remove
like this:
```
typedef struct { 
...
```